cmd/cli/install.go

```diff
@@ -148,17 +150,24 @@ func newInstallCmd(config *helm.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVar(&inst.enableMetricsStack, "enable-metrics-stack", true, "Enable metrics (Prometheus and Grafana) deployment")
 	f.StringVar(&inst.meshName, "mesh-name", defaultMeshName, "name for the new control plane instance")
 	f.BoolVar(&inst.deployJaeger, "deploy-jaeger", true, "Deploy Jaeger in the namespace of the OSM controller")
+	f.StringArrayVar(&inst.setOptions, "set", []string{}, "set values on the command line (can specify multiple or separate values with commas: key1=val1, key2=val2)")
 
 	return cmd
 }
 
 func (i *installCmd) run(config *helm.Configuration) error {
+
+	parsedArgs, err := i.parseSetOptions()
+	if err != nil {
+		return err
+	}
```

Comment:
Can we move this into `resolveValues()` so that wraps all of the value parsing/merging? That would also eliminate the need to pass in an argument which might clean up the tests a bit too.

Reply:
I had originally coded it this way primarily to let `validateOptions()` act on the merged arguments... I have incorporated your comments in a different way to remove the argument and clean up the tests.. please check and let me know..

Reply:
Makes sense. I'm not too concerned with validating options from `--set` at the moment since any value that requires validation has its own flag that users should use instead of `--set`. We can prioritize that if we find people are running into issues because they're using `--set` with invalid values.

@michelleN Would you still like to see `resolveValues()` be a separate step from `validateOptions()`?